confAllChambers.py

```diff
@@ -56,6 +59,8 @@ def launchArgs(shelf,link,slot,run,vt1,vt1bump,config,cName,ztrim,debug=False):
 
     parser.add_option("--config", action="store_true", dest="config",
                       help="Set Configuration from simple txt files", metavar="config")
+    parser.add_option("-l","--latency", type="int", dest = "latency", default = None,
```

Comment:
i would probably also remove the `-l` option, leaving just `--latency`, despite the fact that it may mean someone has to type a little more (i you care about typing, you could change it to `--lat`)

Reply:
Okay, I can change this.

Reply:
Done